### PR TITLE
Add error handling for classes with J9 cp tags

### PR DESCRIPTION
**Pull request by theresa-m**
A CONSTANT_Package_info structure is permitted only in the constant pool
of a class file that declares a module, that is, a ClassFile structure
where the access_flags item has the ACC_MODULE flag set. In all other
class files, a CONSTANT_Package_info structure is illegal.

Signed-off-by: Theresa Mammarella <Theresa.T.Mammarella@ibm.com>

### CODE REVIEW

**Comment by genie-openj9** [comment]
Can one of the admins verify this patch?

**Comment by genie-openj9** [comment]
Can one of the admins verify this patch?

**Comment by genie-openj9** [comment]
Can one of the admins verify this patch?

**Comment by theresa-m** [comment]
Thanks @renfeiw. I fixed the conflict.

File: test/Java8andUp/playlist.xml [hunk 1]
```diff
@@ -1532,5 +1532,25 @@
 			<subset>SE90</subset>
 		</subsets>
 	</test>
+	<test>
+		<featureIds>
+			<featureId>134091</featureId>
```
Comment (DanHeidinga) [inline test/Java8andUp/playlist.xml]: Is this ID an internal bug #?  If so, please replace it with an id that's meaningful in the open

File: test/Java9andUp/playlist.xml [hunk 2]
```diff
@@ -181,4 +181,22 @@
 			<subset>SE90</subset>
 		</subsets>
 	</test>
+	<test>
+		<featureIds>
+			<featureId>134092</featureId>
```
Comment (DanHeidinga) [inline test/Java9andUp/playlist.xml]: Same request re meaningful id

File: test/Java8andUp/src/org/openj9/test/constantPoolTags/ConstantPoolTagTests.java [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+	private int getJavaVersion() {
+		return isJava8() ? classFileVersionJava8 : classFileVersionJava9;
+	}
+
+	private boolean isJava8() {
+		return JAVA_VERSION.startsWith("1.8.0");
+	}
+
+	private boolean isJava9() {
+		return JAVA_VERSION.startsWith("1.9.0") || JAVA_VERSION.startsWith("9-internal");
```
Comment (DanHeidinga) [inline test/Java8andUp/src/org/openj9/test/constantPoolTags/ConstantPoolTagTests.java]: Does this test pass with OpenJ9 builds?  I don't expect the `-internal` to be part of the version number long term.

Comment (theresa-m) [reply test/Java8andUp/src/org/openj9/test/constantPoolTags/ConstantPoolTagTests.java]: Yes it passes with OpenJ9 builds. There was a time when this wasn't passing without matching `9-internal` but just confirmed that is no longer the case.

File: test/Java8andUp/playlist.xml (hunk 1 above)
Comment (theresa-m) [reply test/Java8andUp/playlist.xml]: Okay. Talking to @renfeiw about what this should be. There are many other similar <featureIds> in the test code.

File: test/Java8andUp/src/org/openj9/test/constantPoolTags/ConstantPoolTagTests.java (hunk 3 above)
Comment (DanHeidinga) [reply test/Java8andUp/src/org/openj9/test/constantPoolTags/ConstantPoolTagTests.java]: Can we drop the `-internal`?  This is a temporary artifact that will disappear when 9 is complete and will lead to unexpected test failures.

Comment (theresa-m) [reply test/Java8andUp/src/org/openj9/test/constantPoolTags/ConstantPoolTagTests.java]: Yes. It is gone.

File: test/Java8andUp/src/org/openj9/test/constantPoolTags/ConstantPoolTagTests.java [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+	private int getJavaVersion() {
+		return isJava8() ? classFileVersionJava8 : classFileVersionJava9;
+	}
+
+	private boolean isJava8() {
+		return JAVA_VERSION.startsWith("1.8.0");
+	}
+
+	private boolean isJava9() {
+		return JAVA_VERSION.startsWith("1.9.0");
```
Comment (DanHeidinga) [inline test/Java8andUp/src/org/openj9/test/constantPoolTags/ConstantPoolTagTests.java]:
```
$ java -XshowSettings:properties
....
    java.version = 9-internal
....
```
This check will return false on current openj9 builds as version is temporarily `9-internal`.  Eventually, this the version will be `9` so the check here should be  `startsWith("1.9.0" || startsWith("9")`. 

Sorry, I wasn't clear earlier.

**Comment by DanHeidinga** [comment]
Jenkins test sanity

**Pull request merged**